.gitignore

```diff
@@ -107,5 +107,3 @@ typings/
 # End of https://www.gitignore.io/api/node,visualstudiocode
 
 # Custom rules (everything added below won't be overriden by 'Generate .gitignore File' if you use 'Update' option)
-
-dist
```

Comment:
This needs to be excluded again, not sure why it caused you issues 🤔

Reply:
Oups, I forgot that I created this MR and pushed some last-minute fixes required for our internal build system.